libs/SalesforceSDK/src/com/salesforce/androidsdk/accounts/UserAccountManager.java

```diff
@@ -220,10 +220,11 @@ public Account getCurrentAccount() {
         	if (account != null) {
 
         		// Reads the user ID and org ID from account manager.
+				final String encryptionKey = SalesforceSDKManager.getEncryptionKey();
```

Comment:
Should encryptionKey be a string a better typed object?

Reply:
Could be. I can consider that in the next PR. This PR is just replacing existing methods for now.